.rubocop_todo.yml

```diff
@@ -1,21 +1,20 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-09-11 14:14:48 +0530 using RuboCop version 0.42.0.
+# on 2016-09-24 01:47:19 +0530 using RuboCop version 0.42.0.
```

Comment:
Actually the offenses were related to long method definition in controller action this time. Can't we ignore them? Otherwise I can make some separate private functions that can be called within the actions to shorten them.